_includes/tutorials/aggregating-average/kstreams/code/docker-compose.yml

```diff
@@ -38,13 +38,12 @@ services:
     hostname: schema-registry
     container_name: schema-registry
     depends_on:
-      - zookeeper
       - broker
     ports:
       - "8081:8081"
     environment:
       SCHEMA_REGISTRY_HOST_NAME: schema-registry
-      SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: 'zookeeper:2181'
+      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'PLAINTEXT://broker:9092'
```

Comment:
@awalther28  `PLAINTEXT://` is not required because it reads from `kafkastore.security.protocol` which defaults to PLAINTEXT, https://docs.confluent.io/current/schema-registry/installation/config.html#kafkastore-security-protocol.  If there are multiple SR security types, then this would come into play.  Is the leaning to include `PLAINTEXT://` to be explicit?  The other consideration might be to omit it to lean towards simplicity.

Reply:
@ybyzek I have a leaning to include `PLAINTEXT://` because I don't think it detracts from simplicity and doesn't rely on default values which can change in future versions (albeit changing default security protocols would be pretty disruptive and unlikely 😉 ). This is not a hill I'm willing to die on. I'm okay with changing this to `'broker:9092'` since that is often how the brokers are referenced in other places throughout the repo.

Reply:
Removed plaintext prefix